dashboard/app/views/videos/embed.html.haml

```diff
@@ -5,6 +5,7 @@
 %head
   = stylesheet_link_tag    '/assets/video-js/video-js', media: 'all'
   = render partial: 'layouts/analytics'
+  <script src="https://code.jquery.com/jquery-1.12.4.min.js"   integrity="sha256-ZosEbRLbNQzLpnKIkEdrPv7lOy9C27hHQ+Xp8a4MxAQ="   crossorigin="anonymous"></script>
```

Comment:
Which our version? Right now I think dashboard includes jquery everywhere with the following line
https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/assets/javascripts/application.js.erb#L11